moonshot/integrations/cli/benchmark/cookbook.py

```diff
@@ -340,9 +356,9 @@ def show_cookbook_results(cookbooks, endpoints, cookbook_results, duration):
         console.print("[red]There are no results.[/red]")
 
     # Print run stats
-    console.print(
-        f"{'='*50}\n[blue]Time taken to run: {duration}s[/blue]\n*Overall rating will be the lowest grade that the recipes have in each cookbook\n{'='*50}"
-    )
+    run_stats = f"""{'='*50}\n[blue]Time taken to run: {duration}s[/blue]\n*Overall rating will be the lowest grade
```

Comment:
do we really need this

Reply:
added a variable so it is easier to format the print statement and break them into smaller chunks